packages/darkdump/PKGBUILD

```diff
@@ -0,0 +1,44 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=darkdump
+pkgver=15.f3bc7c0
+pkgrel=1
+pkgdesc='Search The Deep Web Straight From Your Terminal.'
+arch=('any')
+groups=('blackarch' 'blackarch-webapp' 'blackarch-scanner')
+url='https://github.com/josh0xA/darkdump'
+license=('MIT')
+depends=('python' 'python-requests' 'python-termcolor' 'python-argparse')
+makedepends=('git' 'python-setuptools')
+source=($pkgname::"git+https://github.com/josh0xA/darkdump.git")
```

Comment:
If the name of the git repo is already `$pkgname` as in this case you don't need to add `$pkgname::`, use  `source=("git+https://github.com/josh0xA/$pkgname.git")` instead.

Reply:
Ah yes no problem. Will do that moving forward. Updating now.